src/main/java/gov/nasa/jpl/mbee/mdk/mms/sync/manual/ManualSyncRunner.java

```diff
@@ -120,28 +121,41 @@ private File searchForServerElements(Project project, Element element, ProgressS
             nodeIds.add(element.getLocalID());
         }
         File sendData = MMSUtils.createEntityFile(this.getClass(), ContentType.APPLICATION_JSON, nodeIds, MMSUtils.JsonBlobType.SEARCH);
-        HttpRequestBase searchRequest = MMSUtils.prepareEndpointBuilderBasicJsonPostRequest(MMSSearchEndpoint.builder(), project, sendData)
+        //TODO: Return to recursive search by rootElements / ownerID it was crashing MMS4 Elasticsearch
+//        HttpRequestBase searchRequest = MMSUtils.prepareEndpointBuilderBasicJsonPostRequest(MMSSearchEndpoint.builder(), project, sendData)
+//                .addParam(MMSEndpointBuilderConstants.URI_PROJECT_SUFFIX, Converters.getIProjectToIdConverter().apply(project.getPrimaryProject()))
+//                .addParam(MMSEndpointBuilderConstants.URI_REF_SUFFIX, MDUtils.getBranchId(project)).build();
+        HttpRequestBase searchRequest = MMSUtils.prepareEndpointBuilderBasicGet(MMSElementsEndpoint.builder(), project)
```

Comment:
Manual validation can be done at specific elements and/or depths and this change breaks that capability.

Reply:
Unfortunately it is a current limitation of MMS4. We will have to revise in a subsequent patch. We could add logic to allow the depth based case

Reply:
Also it appears the previous implementation using POST was also not passing any kind of depth either so it was already broken

Reply:
Talking with @HuiJun it looks like MMS4 doesnt support depth queries